etl/test/test_extract.py

```diff
@@ -1,30 +1,341 @@
 """
 Tests for the extract.py module
 """
+
+# unit testing modules
 from unittest import TestCase, mock
+from mockito import when, mock as mocki
+# modules referenced during testing
```

Comment:
I am not telling you whether to use mockito or not but there is a requests-mock that we use in other places to mock requests.

Reply:
I understand your consideration. I will look at it.